Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'i18n', '~> 0.6.4'
 gem 'rails-i18n', '~> 3.0.0'
 gem 'turbolinks', '~> 2.5'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
+gem 'roadie', '~> 2.4'
```

Comment:
The current version is 3.0.5, with 3.1.0 in rc.  Could we start with that, or is there something blocking use of 3.x?